src/os/ui/filter/op/isfalseop.js

```diff
@@ -1,21 +1,28 @@
 goog.provide('os.ui.filter.op.IsFalse');
 
 goog.require('os.ui.filter.op.Op');
+goog.require('os.xsd.DataType');
 
 
 /**
  * A 'PropertyIsFalse' operation class.
+ * Based on the OGC Filter Spec
  *
  * @extends {os.ui.filter.op.Op}
  * @constructor
  */
 os.ui.filter.op.IsFalse = function() {
   os.ui.filter.op.IsFalse.base(this, 'constructor',
-      'And', 'is false', 'false', undefined, undefined, 'Supports false, 0, and "false"', 'span', true);
-  this.matchHint = 'false.operation';
+      'And', 'is false', 'false',
+      [os.xsd.DataType.BOOLEAN, os.xsd.DataType.INTEGER, os.xsd.DataType.STRING],
+      'hint="os.ui.filter.op.IsFalse"', 'Supports false, 0, and "false" (case insensitive)', 'span', true);
+  this.matchHint = 'os.ui.filter.op.IsFalse';
```

Comment:
It's generally preferably not to put a dot-delimited class name in for something like this. These are saved to the raw filter XML for the purposes of how we read and decompose filters out in the filter builder, and the class name could change in the future (or we could integrate with other software where the class name is meaningless).

For the other ops, we literally just do things like `this.matchHint = 'like list';`, `'is false'` would be fine here.